scripts/verify-artifact-contents

```diff
@@ -6,7 +6,7 @@ self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
 plugin_version=$(extract_version_from_current_wd)
-plugin_zip=$(ls -1t build/distributions/git-machete-intellij-plugin-${plugin_version}*.zip | head -1)
+plugin_zip=$(./gradlew -q printPluginZipPath | tail -n 1)
```

Comment:
Hmm why is `tail` needed? doesn't `printPluginZipPath` print just a single line? 🤔

Reply:
Well, it does print one single line by itself. But I found this https://www.linkedin.com/posts/klaus-dambeck-69214a_how-to-make-akka-serialization-bulletproof-activity-6973186005717712897-R53h?utm_source=share&utm_medium=member_desktop - which suggests that sometimes there might appear some unexpected warning from Gradle. And that would be printed before our `println`). Seems smart to me.

Reply:
Hmmm this discussion seems to refer to the error that happened after an update to Gradle **4.1**... I doubt the issue is still there in Gradle 7.x that we use. I think it's safe to skip `tail`, I'd never seen a case where that would be needed

Reply:
Ok, I'll skip it then